tests/Domain/Integrations/Controllers/IntegrationControllerTest.php

```diff
@@ -817,7 +817,14 @@ public function test_it_does_not_handle_key_visibility_upgrades_if_unauthorized(
 
     public function test_it_can_show_integration_detail_if_authorized(): void
     {
-        $this->actingAs(UserModel::createSystemUser());
+        $user = UserModel::fromSession([
```

Comment:
Maybe create methods like `actingAsAdmin` and `actingAsIntegrator`?

Reply:
Here you go:
[Use actingAs helper methods](https://github.com/cultuurnet/publiq-platform/pull/989/commits/493aa33e71a93f6e65ac4adf133b298a29a3b570)